test/controllers/clinics_controller_test.rb

```diff
@@ -21,6 +21,16 @@ class ClinicsControllerTest < ActionDispatch::IntegrationTest
     end
   end
 
+  describe 'naf_status' do
+    before {
+      @clinic = create :clinic
+      @clinic[:accepts_naf] = true
+     }
+    it 'should render true' do
```

Comment:
HMU if you have a tough time with this -- rails 5 changed how they did controller tests. http://guides.rubyonrails.org/testing.html#functional-tests-for-your-controllers is the documentation but I spent a lot of quality time with this recently and it's fresh in my mind.